perl.pl

```diff
@@ -0,0 +1,36 @@
+sub ffetch {
+	open("f", "<", $_[0]) or die $!;
+	my @r = <f>;
+	close "f";
+	return @r;
+}
+
+my @hostname = ffetch("/etc/hostname") ;
+my @version = split / /, (ffetch("/proc/version"))[0];
+my @times = split / /, (ffetch("/proc/uptime"))[0];
+my @fi = ffetch("/proc/meminfo");
+
+( my $total ) = ( $fi[0] =~ /(\d+)/ );
+( my $avail ) = ( $fi[2] =~ /(\d+)/ );
+
+opendir my($dh), "/proc" or die $!;
+my @tasks = grep { /^[+-]?\d+$/ } readdir $dh;
+closedir $dh;
+
+my $user = $ENV{'USER'};
+my $term = $ENV{'TERM'};
+my $shell = $ENV{'SHELL'};
+my $tasks = $#tasks + 1;
```

Comment:
i'm not sure if this is correct, but we don't do it in the other files, so remove it for regularized sloc